document-store/src/main/java/org/hypertrace/core/documentstore/postgres/query/v1/vistors/PostgresFilterTypeExpressionVisitor.java

```diff
@@ -66,7 +68,10 @@ public String visit(final RelationalExpression expression) {
   @Override
   public String visit(final KeyExpression expression) {
     return PostgresUtils.prepareParsedNonCompositeFilter(
-        ID, EQ.name(), expression.getKey().toString(), postgresQueryParser.getParamsBuilder());
+        ID,
```

Comment:
Same comment on perf above. I think, if the size of the keys is one, we should use `EQ` operator.

Reply:
Sure. Updated.